mxcube3/routes/queue.py

```diff
@@ -255,37 +248,6 @@ def update_sample(sample_id):
         return Response(status=409)
 
 
-@server.route("/mxcube/api/v0.1/queue/<node_id>/toggle", methods=['PUT'])
```

Comment:
I'd rather that we keep this method, its true that its not used for the moment but it's a existing feature of the queue, and we might want to use it.